### DISCUSSION TITLE
当网站中Page的tags是中文的时候，点击之后会在URL直接展示为中文路径，对SEO并不是很友好，这个有什么办法优化吗

### DISCUSSION THREAD

**Discussion by lkzwc** [facebook/docusaurus#7393] [General]
![image](https://user-images.githubusercontent.com/84896877/167760973-bcc22f43-9938-4f7e-8072-7d487efd22de.png)

**Comment by Josh-Cena** [comment]
You can set the `permalink` for each tag.

```md
---
tags:
  - label: 电信
    permalink: /telecom
---
```

Beware that it will be removed in the future, but there's no other way to do this for now.

**Reply by lkzwc** [reply]
好的 谢谢🙏
